src/mock/filters.js

```diff
@@ -0,0 +1,26 @@
+import { isInPast, isInFuture } from '../utils/filter';
+
+const generateFilters = (points) => {
+  const futurePoints = points.filter((point) => isInFuture(point.dateFrom));
+  const pastPoints = points.filter((point) => isInPast(point.dateFrom));
+
+  return {
+    EVERYTHING: {
+      name: 'everything',
+      isChecked: true,
+      isDisabled: Boolean(points.length === 0)
```

Comment:
Такая запись избыточна, следует заменить на какой то из 2 вариантов
```suggestion
      isDisabled: Boolean(points.length)
      isDisabled: points.length === 0
```